operations/helm/charts/grafana-agent/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: grafana-agent
 description: 'Grafana Agent'
 type: application
-version: 0.8.1
+version: 0.8.2
```

Comment:
IMO this isn't a patch release, since we're not fixing any Helm chart bugs. 

Also, can you update the Helm chart's CHANGELOG?

Reply:
Doesn't everything have to be a patch release to get a unique pullable id? If not I can revert.

Reply:
Oh, sorry, I just mean I think it's a minor release (0.9.0)

Reply:
This is a good chance for me to get more familiar with the tooling :)